tests/test_arcaflow_plugin_utilities.py

```diff
@@ -35,6 +39,13 @@ def test_functional(self):
         self.assertEqual("success", output_id)
         self.assertIsNotNone(output_data.timestamp)
 
+        output_id, output_data = utilities_plugin.wait(
+            params=utilities_plugin.WaitInput(wait_time_ms=500), run_id="plugin_ci"
+        )
+
+        self.assertEqual("success", output_id)
+        self.assertIsNotNone(output_data.waited)
```

Comment:
The output string is simple, why not validate it?

(Although I'm not sure what the point is of returning the requested value, anyway; it seems pointless. Of potential greater value/interest would be the *actual* waited time, though that has complications and probably isn't worth doing.)

Reply:
The existing output is pretty pointless, but I didn't want to output _nothing_ at all.... though maybe _nothing_ is better?

Reply:
If you don't like "nothing" (which I'd think is fine if you've got nothing to say), returning the requested (or actual) time as an `int` (or `float`) would probably be better than a relatively nonsensical string.